### PR TITLE
Add functionality for saying numbers in French

### PR DESCRIPTION
**Pull request by thytrannk**
This pull request adds functionality for saying numbers in French.

### CODE REVIEW

**Comment by minhnhdo** [comment]
Please document which rules you used for saying numbers in French.

**Comment by thytrannk** [comment]
Added. Thanks for the suggestion.

File: src/french.rs [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+static BASES: [&str; 6] = [
+    "mille",
+    "million",
+    "milliard",
+    "billion",
+    "billiard",
+    "trillion",
+];
+
+static BASES_MULT: [&str; 6] = [
```
Comment (minhnhdo) [inline src/french.rs]: It is not very clear what `BASES_MULT` means. Perhaps `BASES_PLURAL` is a better name here?

**Comment by minhnhdo** [comment]
Great! Thanks for your pull request!

**Pull request merged**